### PR TITLE
Improve the layout of edit coach form

### PR DESCRIPTION
**Pull request by rodrigo-morais**
Creating the layout to Edit form.

### CODE REVIEW

**Comment by rodrigo-morais** [comment]
ping @viniciusffj

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Elm SPA example</title>
```
Comment (viniciusffj) [inline src/index.html]: I guess we can use our page title here.

File: src/Coaches/Edit/Edit.elm [hunk 2]
```diff
@@ -1,73 +1,204 @@
...
+          formSpot address model,
+          formName address model,
+          formTypes address model,
+          formCapabilities address model,
+          formDescription address model
         ]
 
 
-formName : Signal.Address Action -> Model -> Html
+formSpot : Signal.Address Action -> Model -> Html.Html
```
Comment (viniciusffj) [inline src/Coaches/Edit/Edit.elm]: Why spot and not spots?

**Comment by rodrigo-morais** [comment]
@viniciusffj, I adjusted the code following your suggestions.
Could you take a look again?

**Pull request closed without merging**